src/scss/_degrees.scss

```diff
@@ -0,0 +1,42 @@
+.degree-promo {
+  border: 1px solid $brand-complementary;
+  display: inline-block;
+  padding: 0;
+  position: relative;
+  vertical-align: middle;
+  width: 100%;
+}
+
+.degree-promo-icon,
+.degree-promo-text {
+  display: inline-block;
+  float: left;
+  font-weight: 400;
+  position: relative;
+  line-height: 1.42857;
+  margin-bottom: -1px;
+  text-align: center;
+}
+
+.degree-promo-icon {
+  background: $brand-complementary;
+  color: $white;
+  font-size: 32px;
```

Comment:
What she said. I was also thinking we should use rems for padding and margins and match what is in Athena when appropriate.